content/en/docs/tasks/administer-cluster/switch-to-evented-pleg.md

```diff
@@ -5,86 +5,91 @@ content_type: task
-* This overhead is exacerbated by Kubelet's parallelism, limiting its scalability
-  and causing poor performance and reliability problems.
-* The goal of `Evented PLEG` is to reduce unnecessary work during inactivity by replacing periodic polling.
+* The current _Generic PLEG_ incurs non-negligible overhead due to frequent polling of container statuses.
+* This overhead is exacerbated by Kubelet's parallelism, limiting its scalability and
+  causing poor performance and reliability problems.
+* The goal of _Evented PLEG_ is to reduce unnecessary work during inactivity
+  by replacing periodic polling.
 
 ## Switching to Evented PLEG
 
 1. Start the Kubelet with the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-   `EventedPLEG` enabled. In Kubelet feature gates can be enabled by editing
-   [config file](/docs/tasks/administer-cluster/kubelet-config-file/) and restarting the Kubelet service.
+   `EventedPLEG` enabled. The kubelet feature gates can be enabled by editing its
+   [config file](/docs/tasks/administer-cluster/kubelet-config-file/) and restarting the kubelet service.
 
-2. Please make sure the node is [drained](/docs/tasks/administer-cluster/safely-drain-node/) before proceeding.
+2. Make sure the node is [drained](/docs/tasks/administer-cluster/safely-drain-node/) before proceeding. 
 
-3. Start the `CRI Runtime` with the `Evented PLEG` support.
+3. Start the `CRI Runtime` with the `Evented PLEG` support. 
 
    {{< tabs name="tab_with_code" >}}
-
-   {{% tab name="containerd" %}}
+   {{< tab name="Containerd" codelang="bash" >}}
    Version 1.7+
-   {{% /tab %}}
-
-   {{% tab name="CRI-O" %}}
+   {{< /tab >}}
+   {{< tab name="CRI-O" codelang="bash" >}}
    Version 1.26+
 
-   Check if the CRI-O is already configured to emit `CRI Events` by verifying the configuration:
+   Check if the CRI-O is already configured to emit CRI Events by verifying the configuration,
 
    ```shell
    crio config | grep enable_pod_events
    ```
 
-   If it is enabled it should show:
+   If it's enabled, the output should be similar to the following:
 
-   ```
+   ```none
    enable_pod_events = true
    ```
 
-   To enable it, start the cri-o daemon with the flag `--enable-pod-events=true` or using a drop in config like:
+   To enable it, start the CRI-O daemon with the flag `--enable-pod-events=true` or
+   use a dropin config with the following lines:
 
-   ```
+   ```ini
    [crio.runtime]
    enable_pod_events: true
    ```
-
-   {{% /tab %}}
+   {{< /tab >}}
    {{< /tabs >}}
 
-   {{< version-check >}}
-
-4. Verify that `Evented PLEG` is in use by looking for the term `EventedPLEG` in the kubelet logs.
+4. Verify that _Evented PLEG_ is in use by looking for the term `EventedPLEG`
+   in the kubelet logs.
 
    The output is similar to this:
 
-   ```
+   ```none
    I0314 11:10:13.909915 1105457 feature_gate.go:249] feature gates: &{map[EventedPLEG:true]}
```

Comment:
Question, if this were
```
   I0314 11:10:13.909915 1105457 feature_gate.go:249] feature gates: &{map[EventedPLEG:false]}
```
then are the instructions clear enough on how to check?

Reply:
No. Clarified this on line 85.